frontend/packages/citizen-frontend/src/applications/editor/verification/ApplicationVerificationView.tsx

```diff
@@ -0,0 +1,133 @@
+  application: ApplicationDetails
+  formData: ApplicationFormData
+  type: ApplicationType
+  closeVerification: () => void
+}
+
+const AttachmentBox = styled.div`
+  border: 2px solid ${espooBrandColors.espooTurquoise};
+  padding: 0 ${defaultMargins.m};
+  display: flex;
+`
+
+const RoundIconStyled = styled(RoundIcon)`
+  margin: ${defaultMargins.m} ${defaultMargins.m} 0 0;
+`
+
+export default React.memo(function ApplicationVerificationViewDaycare({
+  application,
+  formData,
+  type,
+  closeVerification
+}: DaycareApplicationVerificationViewProps) {
+  const t = useTranslation()
+  return (
+    <Container>
+      <ContentArea opaque>
+        <H1>{t.applications.editor.verification.title[type]}</H1>
+        <P
+          dangerouslySetInnerHTML={{
+            __html: t.applications.editor.verification.notYetSent
+          }}
+        />
+        {(formData.serviceNeed.urgent &&
+          formData.serviceNeed.urgencyAttachments.length === 0) ||
+          (formData.serviceNeed.shiftCare &&
+            formData.serviceNeed.shiftCareAttachments.length === 0 && (
+              <AttachmentBox>
+                <RoundIconStyled
+                  content={faInfo}
+                  color={espooBrandColors.espooTurquoise}
+                  size="s"
+                />
+                <div>
+                  <P>
+                    <strong>
+                      {t.applications.editor.verification.attachmentBox.nb}
+                    </strong>
+                    {t.applications.editor.verification.attachmentBox.headline}
+                  </P>
+                  <ul>
+                    {formData.serviceNeed.urgent &&
+                      formData.serviceNeed.urgencyAttachments.length === 0 && (
+                        <li>
+                          {
+                            t.applications.editor.verification.attachmentBox
+                              .urgency
+                          }
+                        </li>
+                      )}
+                    {formData.serviceNeed.shiftCare &&
+                      formData.serviceNeed.shiftCareAttachments.length ===
+                        0 && (
+                        <li>
+                          {
+                            t.applications.editor.verification.attachmentBox
+                              .shiftCare
+                          }
+                        </li>
+                      )}
+                  </ul>
+                  <P>
+                    <a onClick={closeVerification} role="button">
+                      {
+                        t.applications.editor.verification.attachmentBox
+                          .goBackLinkText
+                      }
+                    </a>
+                    {
+                      t.applications.editor.verification.attachmentBox
+                        .goBackLinkText
+                    }
+                  </P>
+                </div>
+              </AttachmentBox>
```

Comment:
and when it did appear the last row text was a bit messed up. Also space missing after "Huom!".

Reply:
![Screenshot 2021-02-03 at 16 35 56](https://user-images.githubusercontent.com/1533854/106764693-b248fd00-6640-11eb-9028-38fff77fc9ca.png)